ios/Podfile.lock

```diff
@@ -381,4 +347,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f72a28dc777229390986fd0b035fd14fbf8c9e3a
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.4.0
```

Comment:
Not critical, when you get some free time, it would be good for you to update local cocoapods to 1.5.3

Reply:
Sure thing :)